src/IdentityManager2/Api/Models/RoleQueryResultResource.cs

```diff
@@ -29,10 +29,8 @@ public RoleQueryResultResource(QueryResult<RoleSummary> result, IUrlHelper url,
         }
     }
 
-    public class RoleQueryResultResourceData : QueryResult<RoleSummary>
+    public class RoleQueryResultResourceData : QueryResult<RoleResultResource>
     {
-        public new IEnumerable<RoleResultResource> Items { get; set; }
```

Comment:
dropping this is needed for System.Text.Json coz it is lost with new overrides :-/